schema/rdl_schema_0.1.json

```diff
@@ -173,6 +212,10 @@
                   "minItems": 1,
                   "uniqueItems": true
                 },
+                "process_type": {
+                  "$ref": "#/$defs/common_process_type",
+                  "description": "Type of hazard process modelled"
+                },
                 "time_start": {
```

Comment:
Remove. Covered by `temporal`. Same for `time_end`, `time_span` and `time_year`.

Reply:
Removed